### ISSUE TITLE
Pages should still load/render even if there is a Ractive parse error

### ISSUE THREAD

**Issue by TehShrike** [ArtskydJ/noddity-render-dom#23]
If you render some markdown like this:

```md
# Sup dawg

{{}
howdy
```

Ractive throws this error, visible in the console:

```
ractive.js:3628 Uncaught ParseError: Expected closing delimiter '}}' after reference at line 4 character 6:
<p>{{}
     ^----
```

This error message should be displayed in the DOM, as the post content.  Probably inside a `<pre>` tag.

Right now it shows up only in the console, and breaks Noddity from running.  The url gets updated, but the content never changes, which is pretty bad UX.

**Comment by saibotsivad** [comment]
Oh yeah, that's a pretty bad deal.

**Comment by saibotsivad** [comment]
What's particularly troubling about this, as it relates to Noddity, is that if you have a *template* which has some syntax error, it appears that this template will not be reloaded (and thereby fixed for the client) until the cache expires (which IIRC you said was ~1 week).

So I think that a proper solution would figure out a way to invalidate the cache for that file, in some way that it will attempt to re-download it sooner than the cache expiration (e.g. try again every few seconds? minutes? backoff algorithm?).

**Comment by TehShrike** [comment]
Yeah, I think that's an accurate read on the situation.  It's rendered non-serious for "post"s because those are reloaded when you visit their page, but for top-level templates or embedded templates, it would be devestating.

I'll write up another issue for this.  It will need to involve noddity-butler

**Comment by TehShrike** [comment]
Created a new issue #24 in this repo, depending on https://github.com/TehShrike/noddity-butler/issues/7.

This particular issue is still a separate thing that needs to be worked (and is a very good stopgap measure I think), but those other issues represent a real problem that should still be solved.
